src/common.ts

```diff
@@ -110,3 +110,19 @@ export type UpdateRegistrantStatusBody = {
 export type UpdateRegistrantStatusParams = {
   occurrence_id?: string;
 };
+
+export type ZoomMeetingRecordingFileType =
+  'MP4' | 'M4A' | 'TIMELINE' | 'TRANSCRIPT' | 'CHAT' | 'CC';
```

Comment:
Looking at the [docs](https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/recordingslist), you may be missing `CSV` from `file_type`.